### PR TITLE
Add dash-table recipe

### PR DESCRIPTION
**Pull request by dkucharc**

### CODE REVIEW

**Comment by dkucharc** [comment]
@conda-forge/help-python This PR is ready for review.

File: recipes/dash-table/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  build:
```
Comment (bgruening) [inline recipes/dash-table/meta.yaml]: This should be the host section.

File: recipes/dash-table/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/dash-table/meta.yaml]: Is setuptools really needed?

File: recipes/dash-table/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - dash_table
+
+about:
+  home: https://dash.plot.ly/datatable
+  dev_url: https://github.com/plotly/dash-table
+  license: MIT
+  license_family: MIT
```
Comment (bgruening) [inline recipes/dash-table/meta.yaml]: can you add a `license_file:`?

**Comment by dkucharc** [comment]
@conda-forge/staged-recipes, @bgruening  All issues raised during the initial review have been addressed.

**Comment by scopatz** [comment]
Thanks @dkucharc!

**Pull request merged**